cadquery/occ_impl/shapes.py

```diff
@@ -809,12 +810,20 @@ def assembleEdges(cls, listOfEdges):
         """
             Attempts to build a wire that consists of the edges in the provided list
             :param cls:
-            :param listOfEdges: a list of Edge objects
+            :param listOfEdges: a list of Edge objects. The edges are not to be consecutive.
             :return: a wire with the edges assembled
         """
+        
         wire_builder = BRepBuilderAPI_MakeWire()
-        for edge in listOfEdges:
-            wire_builder.Add(edge.wrapped)
+        edges_list = TopTools_ListOfShape()
+        for e in listOfEdges:
+            edges_list.Append(e.wrapped) 
+        wire_builder.Add(edges_list)
+        if wire_builder.Error():
```

Comment:
Shouldn't this actually be:
```python
if not wire_builder.IsDone():
```
given that `IsDone` returns a bool and `Error()` returns an enum? I'm basing my question on the OCC docs: https://www.opencascade.com/doc/occt-6.9.0/refman/html/class_b_rep_builder_a_p_i___make_wire.html#a4e60ad05c4fd315e7b8304b8687b9fa3 but I'm not sure if I get why there are two similar methods.

Reply:
Good point! 
The original motivation to have these warnings in the new ```assembleEdges``` was this comment in the OCC docs:
```If some of them are not connected the Status give DisconnectedWire but the "Maker" is Done() and you can get the partial result.```
The warning is supposed to inform the user if the wire is done while he does not get the expected results. That's why ```Error()``` is used rather than ```IsDone()```. Actually I could not come with any example where ```IsDone()``` returns false...
However you are correct that ```if wire_builder.Error():``` is not adequate as ```Error()``` returns a number. I found that the returned BRepBuilderAPI_MakeWire::Error() values are
    BRepBuilderAPI_WireDone = 0
    BRepBuilderAPI_EmptyWire = 1
    BRepBuilderAPI_DisconnectedWire = 2
    BRepBuilderAPI_NonManifoldWire = 3
Therefore I have now replaced ```if wire_builder.Error():``` with ```if wire_builder.Error()!=0:``` and modified the first test to pass an open wire to ```assembleEdges``` so that ```IsDone``` is still true but ```Error``` returns 2, so that the warning functionality is tested. I have also added the signification of the 4 values in the docstring.
What do you think?

Reply:
Looks good to me.